packages/orbit-tailwind-preset/src/presets/foundation/cssVarsFoundation.ts

```diff
@@ -1,102 +1,111 @@
-import { defaultTokens } from "@kiwicom/orbit-design-tokens";
+import { defaultTokens, parseToRgba } from "@kiwicom/orbit-design-tokens";
+
+const getChannels = (color: string) => parseToRgba(color).slice(0, -1).join(", ");
+const rgba = (color: string, fallback: string) =>
```

Comment:
Nitpick and non blocking suggestion:
I was confused about this when reading the code below, on the class definition. Because I though you were using directly the css property `rgba` but then you were using the string directly, instead of the `var(--`. Only after I found out that was actually this function, so an abstraction.

My suggestion is to rename this function for better and undoubtable reading of the code. I wouldn't mind calling it `generateRGBA`, but up to you.

Reply:
yeah, good suggestion, thanks